dynamic_array.py

```diff
@@ -0,0 +1,41 @@
+# Hacker Rank Problem Link: https://www.hackerrank.com/challenges/dynamic-array/problem
+
+#!/bin/python3
+
+import os
+import sys
+
+#
+# Complete the dynamicArray function below.
+#
+def dynamicArray(n, queries):
+    #
+    # Write your code here.
+    #
+    lastAns = 0
+    seqList = [[]]* n
+    for query in queries:
+        if query[0] == 1:
+            seqIndex  = (query[1] ^ lastAns) % n
+            seqList[seqIndex] = seqList[seqIndex] + [query[2]]
+        elif query[0] == 2:
+            seqIndex = (query[1] ^ lastAns) % n

```

Comment:
seqIndex is duplicated in both branches. This should be refactored into a common variable for both.

Extra points for having a small function to handle this.